src/app/modules/time-clock/pages/time-clock.component.ts

```diff
@@ -2,7 +2,9 @@ import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { ActionsSubject, select, Store } from '@ngrx/store';
 import { ToastrService } from 'ngx-toastr';
 import { Subscription } from 'rxjs';
-import { filter } from 'rxjs/operators';
+
+import { filter, map } from 'rxjs/operators';
+import { threadId } from 'worker_threads';
```

Comment:
are we using this two imports ? the `import { filter, map } from 'rxjs/operators';` and the `import { threadId } from 'worker_threads';`